### ISSUE TITLE
[Bug][a11y]: Add aria-orientation for resizable split-view when using vertical orientation

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3386]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Split-view

### Expected behavior

By default all separators have an implicit orientation of horizontal. But for vertical separator, aria-orientation="vertical" needs to be added.

Note: 

Keep in mind that the terminology used in this component for what is horizontal and vertical is a little confusing. In the documentation, where [vertical resizable example](https://opensource.adobe.com/spectrum-web-components/components/split-view/#vertical-resizable-%26-collapsible) was shown, the separator here is actually a horizontal separator and doesn't need aria-orientation because it has an implicit orientation of horizontal by default. 

Example for [Horizontal resizable](https://opensource.adobe.com/spectrum-web-components/components/split-view/#horizontal-resizable-%26-collapsible) actually shows a vertical separator. For these, aria-orientation="vertical" should be added.

### Actual behavior

Orientation is not communicated to screenreader users.
